docs/catalog/cloud/use-compiled-languages.md

```diff
@@ -0,0 +1,44 @@
+---
+version: 1.0
+submitted_by: markus-ntt-seidl
+published_date: TBD
+category: cloud
+tags: 
+ - cloud
+ - role:cloud-engineer
+ - size:small
+---
+
+# Use compiled languages
+
+## Description
+
+Interpreted languages need to be parsed, compiled and executed when the application starts or a workload arrives. This tends to be more energy heavy then when a compiled language is used. The compilation is then only done once, saving on resources.
+
+## Solution
+
+Use compiled languages whenever possible or compile interpreted languages.
+
+## SCI Impact
+
+`SCI = (E * I) + M per R`  
+[Software Carbon Intensity Spec](https://grnsft.org/sci)
+
+Concerning the SCI equation, use compiled languages will impact as follows:
+
+- `E`: The embodied carbon emissions will be reduced, as compiled binaries tend to be smaller then the sources they are compiled from
+- `I`: Running compiled binaries is more energy efficient and uses less energy which outweighs the energy consumed in compiling it to binary upfront.
```

Comment:
Ahh, maybe this should be `E` instead of `I` :-). Nit: We try to keep the SCI references in order of the equation so E comes first (if applicable), then I, then M, then R.

Reply:
I did not re-order the template:

- `E`: [PATTERN_ENERGY_SCI_IMPACT]
- `I`: [PATTERN_INTENSITY_SCI_IMPACT]
- `M`: [PATTERN_EMBODIED_SCI_IMPACT]
- `R`: [PATTERN_RATE_SCI_IMPACT]

Did i miss something, as the order seems as you describe it :)